webapp/components/integrations/components/installed_outgoing_webhooks.jsx

```diff
@@ -115,6 +115,7 @@ export default class InstalledOutgoingWebhooks extends React.Component {
                     creator={this.state.users[outgoingWebhook.creator_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
Similar thing here to @jwilander's on your other PR. `installed_incoming_webhook.jsx` should be moved into `installed_incoming_webhook/installed_incoming_webhook.jsx` and then you should create a container in  `installed_incoming_webhook/index.jsx` that connects to the redux store to get the channel.